src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: string;
+  description: string;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={`${name} avg`}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={formatDelayFooterTitle(metric.meta.value as number[], metric.meta.type)}
```

Comment:
What we want to show in the footer depends on the metric. For event loop delay, it's the percentiles, for the Load it's the load average intervals and in the case of the response times, we're showing the max to give more context to what the mean is.

Reply:
Do you need to cast `metric.meta.value as number[]` if you do something like `metric.meta?.value && (<MetricCardFooter...` ?

> We could, theoretically, do all the formatting in `load_status` but I chose to do the formatting in the component because it feels more relevant here.

Yeah we were mixing concerns here already anyway -- some formatting happens in `load_status`, some in the component. It would be nice to have `metric` preformatted so that you don't need any conditional logic in the component, but I don't have particularly strong feelings on it.

Reply:
Value is `value: number | number[];`, so it has to be force-casted.

I think we should really get rid of this `formatMetrics` function that converts a properly typed object from the server to this untyped `Metric` array. It maybe made sense previously as we wanted to have a generic component to ~rule them all~ handle all metrics in the same way, but now that we have specific display and component depending on the metric, I think `formatMetrics` is just making things worse. 

That's quite a lot of additional changes though, so we probably don't want to do this in current PR.

Reply:
> That's quite a lot of additional changes though, so we probably don't want to do this in current PR.

We want this to go into 7.17 as a bug fix (forgetting to add the `event_loop_delay` metrics), so keeping the changes in this PR small is the way to go. It's "only" adding the metrics we didn't have, hence, fixing a bug.

I have the feeling the status page is going to evolve a lot and probably (hopefully) soon to add more detail to the status info at least. We'll need to get design in here to help us figure out the best way we can make this page more useful for diagnosing issues. Once that redesign is done, we can go ahead and refactor.

Reply:
@lukeelmers @pgayvallet I've made the changes and errored on the side of caution, keeping the changes very small. Would you mind having another look please?